lib/galaxy/managers/interactivetool.py

```diff
@@ -281,10 +281,11 @@ def get_entry_point_path(self, trans, entry_point):
         entry_point_prefix = self.app.config.interactivetools_prefix
         rval = "/"
         if not entry_point.requires_domain:
+            rval = str(urlparse(self.app.config.galaxy_infrastructure_url).path).rstrip("/").lstrip("/")
```

Comment:
This is pretty explicitly not meant to be used this way. It is meant as a URL for Galaxy internal to the network - not for external users. That is why it is called infrastructure URL. I don't know that I have a good alternative though.

I mean I guess the right thing to do is to persist the URL Galaxy was accessed via when the job was created with in the database and then use this abstraction (xref https://github.com/galaxyproject/galaxy/pull/11904/files)- this would allow jobs to reason about these things and handle multiple Galaxy front end servers all pointing at the same backend (like the EU flavors subdomain setup).

That is probably too big a fix for this. Can you just add a configuration option for prefix - ``app.config.interactivetools_proxy_url_base`` or something?